tests/resources/test_derive_mirror_panel_rnda.yml

```diff
@@ -0,0 +1,25 @@
+---
+mirror_list:
+  class: File
+  description: |
+    Mirror list with including focal lengths per mirror panel.
+  location:
+    https://raw.githubusercontent.com/gammasim/simtools/main/tests/resources/MLTdata-preproduction.ecsv
+mirror_panel_psf_measurement:
```

Comment:
Not a real issue but why do we assume the PSF measurements are in a separate file than the mirror list with the focal lengths? It would be safer to have them all in the same file, no?

Reply:
Yes, this is how it is implemented in the simtool. These are different measurements, or? This was the motivation to separate it.

Reply:
Yes you are right, they are different measurements (or at least can be). I guess it's a good enough reason to keep them separate. I will approve now then.